pkgs/applications/video/obs-studio/plugins/obs-gstreamer.nix

```diff
@@ -10,18 +10,37 @@
 
 stdenv.mkDerivation rec {
   pname = "obs-gstreamer";
-  version = "0.3.3";
+  version = "0.3.4";
 
   src = fetchFromGitHub {
     owner = "fzwoch";
     repo = "obs-gstreamer";
     rev = "v${version}";
-    hash = "sha256-KhSBZcV2yILTf5+aNoYWDfNwPiJoyYPeIOQMDFvOusg=";
+    hash = "sha256-CDtWe4bx1M06nfqvVmIZaLQoKAsXFnG0Xy/mhiSbMgU=";
   };
 
   nativeBuildInputs = [ pkg-config meson ninja ];
   buildInputs = with gst_all_1; [ gstreamer gst-plugins-base obs-studio ];
 
+  # - We need ".out" instead of default derivation, otherwise it brings gstreamer-bin;
+  # - without gst-plugins-base it won't even show proper errors in logs;
+  # - Without gst-plugins-bad it won't find element "h264parse";
+  # - gst-vaapi adds "VA-API" to "Encoder type";
+  # - gst-plugins-ugly adds "x264" to "Encoder type";
+  # Tip: "could not link appsrc to videoconvert1" can mean a lot of things, enable GST_DEBUG=2 for help.
+  passthru.obsWrapperArguments =
+    let
+      gstreamerHook = v: "--prefix GST_PLUGIN_SYSTEM_PATH_1_0 : ${v.out}/lib/gstreamer-1.0";
```

Comment:
Does this work?
```suggestion
      gstreamerHook = v: "--prefix GST_PLUGIN_SYSTEM_PATH_1_0 : ${lib.getLib v}/lib/gstreamer-1.0";
```

Reply:
Sorry, I didn't know about `lib.get*`. Thanks for the info.